packages/dev/core/src/Engines/Extensions/engine.renderTarget.ts

```diff
@@ -226,14 +226,14 @@ ThinEngine.prototype.updateRenderTargetTextureSampleCount = function (rtWrapper:
         return 1;
     }
 
+    samples = Math.min(samples, this.getCaps().maxMSAASamples);

```

Comment:
I would separate this into another PR. These two places are not the only place that does this. We need to make sure they are consistent across the entire code base.

Reply:
Makes sense.  I'll rollback the changes to this line (and all related instances) and handle them in a separate PR.